packages/tables/backend/sirius-components-collaborative-tables/src/main/java/org/eclipse/sirius/components/collaborative/tables/handlers/EditCellEventHandler.java

```diff
@@ -92,18 +90,24 @@ public void handle(One<IPayload> payloadSink, Many<ChangeDescription> changeDesc
         String message = this.messageService.invalidInput(tableInput.getClass().getSimpleName(), EditCheckboxCellInput.class.getSimpleName());
         IPayload payload = new ErrorPayload(tableInput.id(), message);
 
-        if (tableInput instanceof EditCheckboxCellInput editCheckboxCellInput) {
-            var optCell = this.tableQueryService.findCellById(table, UUID.fromString(editCheckboxCellInput.cellId()));
+        if (tableInput instanceof IEditCellInput editCellInput) {
+            var optCell = this.tableQueryService.findCellById(table, UUID.fromString(editCellInput.cellId()));
```

Comment:
Why UUID.fromString like that? Either the cellId is a String or it's an UUID but there's something fishy here.